public/articles/getarticles.json

```diff
@@ -5,6 +5,8 @@
       "id": 342432,
       "title": "Why should you not use ReactJS and move your project to Angular as soon as possible",
       "author": "John Doe",
+      "authorprofile": "/user/4324211",
```

Comment:
create an object as `author` which should be like this

```json
"author":{
    "id": "4324211",
    "name": "John Doe",
    "profileImg": "https://upload.wikimedia.org/wikipedia/commons/4/4a/GitHub_Mark.png"
}
```

this `author` will be nested inside each article